### ISSUE TITLE
[DataCap Application] <Duitang> - <Arts&Recreation>3

### ISSUE THREAD

**Issue by Mark-duitang** [ipfsforcezuofu/ipfsforce-allocator#30]
### Data Owner Name

Duitang Information Technology (Shanghai) Co., Ltd.

### Data Owner Country/Region

China

### Data Owner Industry

Arts & Recreation

### Website

https://www.duitang.com/

### Social Media Handle

https://weibo.com/duitang/

### Social Media Type

WeChat

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

10PiB

### Expected size of single dataset (one copy)

2.5PiB

### Number of replicas to store

4

### Weekly allocation of DataCap requested

1.25PiB

### On-chain address for first allocation

f3qgtbnk6h3dfg6zwkqwddez6rj7lvrcvurvltuk3q6oqsnui4ptcp5adrwddy5nchopjs7r5ple5rylofbvpq

### Data Type of Application

Public, Open Commercial/Enterprise

### Custom multisig

- [X] Use Custom Multisig

### Identifier

No response

### Share a brief history of your project and organization

```text
Duitang.com is an interest community focused on sharing aesthetic wallpapers, gathering tens of millions of wallpaper enthusiasts and creators. Here, users can find a wide variety of high-definition pictures, including popular avatars, landscape wallpapers, girl backgrounds, anime character wallpapers, etc., to meet the personalized needs of different users.

As a community platform centered on aesthetic wallpapers, the content within the community covers multiple fields, from wallpapers of popular IPs to works of original designers, from natural landscape photography to the reinterpretation of anime illustrations. Users can not only browse and download these aesthetic wallpapers, but also interact and communicate with other users, and even upload their own works to share their creativity and talents with more people.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### If answered yes, what are the other projects/ecosystem stakeholders

```text
No response
```

### Describe the data being stored onto Filecoin

```text
Community HD wallpapers, gifs, videos, etc.
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you answered "Other" in the previous question, enter the details here

```text
No response
```

### If you are a data preparer. What is your location (Country/Region)

China

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
Graphsplit
```

### If you are not preparing the data, who will prepare the data?  (Provide name and business)

```text
No response
```

### Has this dataset been stored on the Filecoin network before? If so, please explain and make the case why you would like to store this dataset again to the network. Provide details on preparation and/or SP distribution.

```text
We have not yet stored our dataset on the Filecoin network.
```

### Please share a sample of the data

```text
https://www.duitang.com/blog/?id=1461736982
https://www.duitang.com/blog/?id=1491490640
https://www.duitang.com/blog/?id=1496113494
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### If you chose not to confirm, what was the reason

```text
No response
```

### What is the expected retrieval frequency for this data

Yearly

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, Europe

### How will you be distributing your data to storage providers

HTTP or FTP server, IPFS, Others

### How did you find your storage providers

Slack, Big Data Exchange, Partners

### If you answered "Others" in the previous question, what is the tool or platform you used

```text
No response
```

### Please list the provider IDs and location of the storage providers you will be working with.

```text
1，f03224648， Singapore
2，f03137648,  New York, USA
3，f03161261， USA/New York
4，f03187925， USA
```

### How do you plan to make deals to your storage providers

Boost client, Lotus client, Droplet client, Singularity

### If you answered "Others/custom tool" in the previous question, enter the details here

```text
No response
```

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
Thank you for your reply. I have sent the proof materials to your email address. Please check it. Thank you!

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
I can accept that, thank you very much!

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
OK, thanks for the advance notice, and we have another SP: f03224648 reporting that their remaining storage is running out and may need to be suspended for a while, so we decided that when f03224648 stops cooperating, we will add a new SP: f03136895. Region: US/Virginia. I will send you the KYC information via email, thank you!

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
I have synchronized SP:f03224648 to you two days ago. It will be replaced by SP:f03136895 after the storage is used up. However, SP:f03136895 is temporarily uncooperative due to its own operational issues, so we have found a new SP:f03087446. I will send you the relevant email. I am sorry to bother you.

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
Thank you for providing us with 1P last time. According to your allocation, we hope to continue to provide 1P in the next round. We will add two SPs. I will email you to add KYC information later:
f03157902 Region: US/Los Angeles
f03027241 Region: US/New York
This round we plan to cooperate with these four SPs, 250T per SP, 4 copies
f03187925
f03087446
f03157902
f03027241

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
Hi, I have just sent you an email, could you please approve my application? Thank you!

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
That's great. I've seen the problem you mentioned. I have applied under other distributors. The reason is that a single distributor can only supply a limited amount of data continuously. In order to meet my data storage speed and volume, we applied separately. I will change the application procedure description for the specific situation and the SPs that will cooperate in this round.
Because it has been a long time, the SPs we cooperate with have changed. I have also sent the latest SP's missing KYC to your email. Please check it. Please help us approve the next round of 1.25P (4 SPs each with 320T). Thank you very much for your hard work.

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
The SP f03136895 search procedures are all normal. I have tested many rootCid searches and there are no problems. There are no search failure and error logs in our logs. Maybe the time is too short to search. Could you please help me approve the next round? Thank you very much!
![d55f028bb26ad0f6d3edb60b776ffa2](https://github.com/user-attachments/assets/4eb946ae-9e2c-4bbb-94db-452ad6dd3ca3)
![52872f36de0bf0d53bb8547eff83a44](https://github.com/user-attachments/assets/f371ddf4-c8fc-40bd-853b-c30c09f21505)

**Comment by jinyexon** [comment]
checker:manualTrigger

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
We have used up all of it. Can we approve the next round?

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
Yes, we are paying attention to this issue, but we haven't found the cause of the problem yet. We have checked the search procedures and they are all normal. I am trying to re-post the rootcid I sent before and search again to see if there is any improvement. If there is still no improvement, I can only trouble you to ask the official to provide the search error log.

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
Hello, I am willing to cooperate with your work. Please refer to the following points:
1. Number of copies: We have always stored 4 copies. 9 is the number of SPs, not the number of stored copies. Due to the SP's own operation and insufficient pledged coins, we will replace the SP midway.
2. About VPN: I asked the SP, and they said they did not use the VPN service. How do you judge that the SP used the VPN service? Can you provide some evidence to facilitate communication between me and the SP?
3. Data preparation and data set content:
For the case of inconsistent file sizes, we split and merge the original data sets and finally package the data files. Data content: We are a picture sharing community (covering 27 categories such as movies, wallpapers, avatars, emoticons, animated pictures, weddings, food, pets, etc.), with billions of high-definition high-quality pictures.
4. Obtained from the other two allcators：
Since allcators are often in short supply, we do not want to have storage interruptions, so we have made multiple applications.

**Comment by Mark-duitang** [comment]
@filecoin-watchdog 
Thank you very much for your further explanation. We are replying to you so late because we are conducting self-examination and have now found out the problem:
1. After investigation, we found that the main reason was that since November, due to SP's own operational problems, we replaced some SPs, which caused some errors in code logic. We originally divided the data equally into 4 parts. After replacing the SP in the middle, the newly added node will regenerate the replaced node data once, so more than 4 nodes will use duplicate data. If only 4 SPs are added by default without changing in the middle, this situation will not occur. We will change the code logic later, and the newly added SP will continue from the data generated by the replaced SP instead of generating it again, which can avoid this problem;
2. We checked the database and found that there were indeed human errors in the past, which caused 1590 messages to have repeated CIDs for a total of 5 times;
For these two errors, we will make technical adjustments in the future to avoid them. Thank you again for helping us find the problem in time.

**Comment by Mark-duitang** [comment]
@ipfsforcezuofu 
Regarding the VPN issue, it is difficult for us to completely prevent SP from having such situations because they have promised not to use it and it is difficult for us to confirm that they are using VPN. However, we are very willing to cooperate to do more communication and ask SP to cooperate with rectification. My plan for the current situation is that we will cancel the current SP cooperation in the next round and look for a new SP. Thank you!
